File: 251011/짝수의 합 n번 구하기/find-the-sum-of-even-numbers-n-times.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    cin >> a;

    int b, c;

    for(int i = 0; i < a; i++) {
        int sum = 0;
        cin >> b >> c;
        for(int j = b; j <= c; j++) {
            if(j % 2 == 0) {
                sum += j;
            }
        }
        cout << sum << "\n";
    }
    return 0;
}

File: 251114/짝수의 합 n번 구하기/find-the-sum-of-even-numbers-n-times.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a = int.Parse(Console.ReadLine());
        for(int i = 1; i <= a; i++) {
            var input = Console.ReadLine().Split();
            int b = int.Parse(input[0]);
            int c = int.Parse(input[1]);
            int count = 0;

            for(int j = b; j <= c; j++) {
                if(j % 2 == 0) {
                    count += j;
                }
            }
            Console.WriteLine(count);
        }
    }
}
